my-sms/spec/models/student_spec.rb

```diff
@@ -1,5 +1,66 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 
-RSpec.describe Student, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe Student, type: :model do
+  it 'has a valid factory' do
+    expect(FactoryBot.build(:student)).to be_valid
+  end
+
+  it 'has a factory that generates unique emails' do
+    student = FactoryBot.build(:student)
```

Comment:
As with the other specs, I'd recommend pulling the assignment out of the block to something like the following:

```
let(:student) { FactoryBot.build(:student) }
let(:other_student) { FactoryBot.build(:student) }

it 'has a factory that generates unique emails' do
  expect(student.email).not_to eq other_student.email
end
```

Also, I'm not suggesting you delete this, but we don't tend to test our factory behaviour. It's not a terrible idea though.

Reply:
Ended up installing shoulda-matchers as well and making the whole thing cleaner.
f45e16ad5f92b912d1aeee870101c3a8026d406f

Reply:
Great idea. I coulda mentioned that 👍